contracts/crowdsale/emission/MintedCrowdsale.sol

```diff
@@ -0,0 +1,11 @@
+pragma solidity ^0.4.18;
+
+import "../Crowdsale.sol";
+import "../../token/ERC20/MintableToken.sol";
+
+contract MintedCrowdsale is Crowdsale {
+
+  function _emitTokens(address _beneficiary, uint256 _tokenAmount) internal {
+    require(MintableToken(token).mint(_beneficiary, _tokenAmount));
+  }
+}
```

Comment:
Do we want a function to recover the ownership of the token after the crowdsale has ended? Although that would incur into the notion of _end_.

Reply:
I'd also add a comment explaining that to `mint`, this crowdsale needs to be the `MintableToken`'s owner.

Reply:
We can model it as a new crowdsale flavor indeed